child/child.go

```diff
@@ -353,7 +358,18 @@ func (c *Child) signal(s os.Signal) error {
 	if !c.running() {
 		return nil
 	}
-	return c.cmd.Process.Signal(s)
+
+	sig, ok := s.(syscall.Signal)
+	if !ok {
+		return fmt.Errorf("[ERR] (child) bad signal: %s", s)
```

Comment:
This is formatted more as of a log line than an error message. Can you update it?

Reply:
Missed that in a refactor. Thanks.